cypress/e2e/tests/migration/applicationinventory/applications/filter.test.ts

```diff
@@ -339,4 +332,30 @@ describe(["@tier2"], "Application inventory filter validations", function () {
         deleteByList(applicationsList);
         deleteByList(stakeholders);
     });
+    const filterApplicationsBySubstring = () => {
```

Comment:
if you want to create a function, create it outside of the describe block, and provide params and return type

Check: `cypress/e2e/tests/migration/migration-waves/unlink_application.test.ts` for an example

Reply:
added void as return type, no params passed as i think it is may not needed. and moved it outside of describe block

Reply:
maybe i could move the substrings as a global consts but i think unnecessary since this is the only TC for it